sys/src/9/386/rtl8139.c

```diff
@@ -0,0 +1,1418 @@
+/* rtl8139.c: A RealTek RTL8129/8139 Fast Ethernet driver for Linux. */
```

Comment:
we already have an 8139 driver. And 8139 are not wireless. I'm not sure what this is?